neptune-tinkerpop/.gitignore

```diff
@@ -1,2 +1,3 @@
 *.class
 target/
+*.kryo
```

Comment:
Out of curiosity - did you end up seeing some `*.kryo` files locally? 

Reason I'm asking - we did use kryo to support [persistence for the old StepFunctions provider](https://github.com/localstack/localstack-artifacts/blob/master/stepfunctions-local-patch/cloud/localstack/PersistenceContext.java#L35-L61), prior to releasing our StepFunctions v2 rewrite in pure Python. Was just curious if Gremlin also uses kryo - and seems like it, according to [the docs here](https://tinkerpop.apache.org/docs/current/reference/#gryo).

Reply:
I did, when creating the maven package, a file gets created at `./neptune-tinkerpop/3.6.2/file:/home/matt/.m2/repository/org/apache/tinkerpop/tinkergraph-gremlin/3.6.2/test-case-data/TinkerGraph/resources/tinkerpop-modern-v3d0.kryo`.